### DISCUSSION TITLE
is there a way to load the api key in runtime?

### DISCUSSION THREAD

**Discussion by valn1** [react-native-maps/react-native-maps#4867] [Feature requests]
the app i am developing is supposed to work by using the user's api key instead of ours.
we get it from a request that is made after the user logs in, but this lib seems to not have any way of setting the api key

**Comment by recallwei** [comment]
I face the same problem. Seems that storing the API key directly is not secure.

**Comment by valn1** [comment]
from what i saw, even if they wanted to, they could not do it as the google services api itself needs the key in buildtime.
my recommendation is to try a different lib, i went with mapbox, but there is open street maps and some others that are free until a certain amount of requests
